### PR TITLE
Add filtering to deseq2 process

### PR DESCRIPTION
**Pull request by JenkoB**

### CODE REVIEW

File: resolwe_bio/processes/differential_expression/deseq.yml [hunk 1]
```diff
@@ -23,6 +23,12 @@
       type: list:data:expression
       description: >
         Control samples (replicates)
+    - name: filter
+      label: Filtering of raw counts
+      type: basic:integer
+      default: 10
+      description: >
+        The cut of value for filtering of raw counts
```
Comment (mstajdohar) [inline resolwe_bio/processes/differential_expression/deseq.yml]: cutoff -> https://en.wikipedia.org/wiki/Cutoff

Comment (JenkoB) [inline resolwe_bio/processes/differential_expression/deseq.yml]: I changed to: The threshold value for raw counts filtering.

File: resolwe_bio/processes/differential_expression/deseq.yml [hunk 2]
```diff
@@ -23,6 +23,12 @@
       type: list:data:expression
       description: >
         Control samples (replicates)
+    - name: filter
+      label: Filtering of raw counts
```
Comment (mstajdohar) [inline resolwe_bio/processes/differential_expression/deseq.yml]: Write what is the parameter, not what is does. Think of this as along parameter name. For example, "Raw counts threshold" or "Raw counts filtering threshold".

File: resolwe_bio/processes/differential_expression/deseq.yml [hunk 3]
```diff
@@ -23,6 +23,12 @@
...
+      type: basic:integer
+      default: 10
+      description: >
+        The threshold value for raw counts filtering.
```
Comment (mstajdohar) [inline resolwe_bio/processes/differential_expression/deseq.yml]: Write what it does. For example, "Filter genes in the expression matrix input. Remove genes where the number of counts in all samples is below the threshold.".

**Comment by mstajdohar** [comment]
Also, rebase on to master.

**Comment by dblenkus** [comment]
Fixed, rebased and (again) bumped process version.

**Pull request merged**